test/lit/passes/dae-optimizing.wast

```diff
@@ -16,29 +16,19 @@
  (global $global$0 (mut i32) (i32.const 10))
  ;; CHECK:      (func $0 (result i32)
  ;; CHECK-NEXT:  (local $0 i32)
- ;; CHECK-NEXT:  (local $1 i32)
  ;; CHECK-NEXT:  (drop
  ;; CHECK-NEXT:   (if (result f32)
  ;; CHECK-NEXT:    (local.tee $0
  ;; CHECK-NEXT:     (i32.const 33554432)
  ;; CHECK-NEXT:    )
- ;; CHECK-NEXT:    (loop $label$2 (result f32)
+ ;; CHECK-NEXT:    (block (result f32)
  ;; CHECK-NEXT:     (if
  ;; CHECK-NEXT:      (global.get $global$0)
  ;; CHECK-NEXT:      (return
  ;; CHECK-NEXT:       (local.get $0)
  ;; CHECK-NEXT:      )
  ;; CHECK-NEXT:     )
- ;; CHECK-NEXT:     (local.set $0
- ;; CHECK-NEXT:      (local.get $1)
- ;; CHECK-NEXT:     )
- ;; CHECK-NEXT:     (local.set $1
- ;; CHECK-NEXT:      (i32.const 0)
- ;; CHECK-NEXT:     )
- ;; CHECK-NEXT:     (br_if $label$2
- ;; CHECK-NEXT:      (local.get $0)
- ;; CHECK-NEXT:     )
- ;; CHECK-NEXT:     (f32.const 1)
+ ;; CHECK-NEXT:     (f32.const 0)
```

Comment:
This seems odd - it used to return 1, and now returns 0?

Reply:
Because this value is drop, so return whatever is Ok, just for keep align in true and false branch.
I have update the testcase and received this value by a global. The return value is 1 now.